### DISCUSSION TITLE
x10drvrules.ino

### DISCUSSION THREAD

**Discussion by Tino4** [benzino77/tasmocompiler#269] [Ideas]
Hi guys , in the file x10drvrules.ino there is a once flag execution code for executing a rule only one time.Maybe someone can merge a extra case in this file for executing a rule twice or three times?? Its just a tought.Thanks in advance.Best Regards, Tino

**Comment by benzino77** [comment]
Hi there,
I'm not sure this is a good place to ask such question. TasmoCompiler is just a tool to build/compile Tasmota firmware. I think you should ask guys from Tasmota Dev The best place to ask such question will be Tasmota Discord Channel.

**Comment by Tino4** [comment]
Okay thanks for the info, best regards TinoVerzonden vanaf mijn telefoon-------- Oorspronkelijk bericht --------Van: benzino77 ***@***.***>Datum: za 16 apr. 2022 16:46Aan: benzino77/tasmocompiler ***@***.***>Cc: Tino4 ***@***.***>, Author ***@***.***>Onderwerp: Re: [benzino77/tasmocompiler] x10drvrules.ino (Discussion #269)
Hi there,
I'm not sure this is a good place to ask such question. TasmoCompiler is just a tool to build/compile Tasmota firmware. I think you should ask guys from Tasmota Dev The best place to ask such question will be Tasmota Discord Channel.
